### DISCUSSION TITLE
which version of python?

### DISCUSSION THREAD

**Discussion by Megsman** [alejandroautalan/pygubu-designer#191] [General]
Hi I can't seem to make it work in my 3.8.16, 3.10.9 and 3.11.1 python environments. Which version works best for pygubu-designer?

using MacOS Monterey.

Thanks.

**Comment by Megsman** [comment]
I just tried it on python 3.9.16 and it works. I also had to "brew install python-tk@3.9"

**Comment by alejandroautalan** [comment]
Hello  @Megsman thanks for trying pygubu.

> Hi I can't seem to make it work in my 3.8.16, 3.10.9 and 3.11.1 python environments. Which version works best for pygubu-designer?
> using MacOS Monterey.

It should work with any python version from 3.6 to 3.11.  I tested it only on MacOS Mojave and Catalina, using the python installer from www.python.org

> I just tried it on python 3.9.16 and it works. I also had to "brew install python-tk@3.9"

I didn't know this, thanks for the info.

Regards.
Alejandro A
